### ISSUE TITLE
version numbers on macOS are damaged. 

### ISSUE THREAD

**Issue by core-code** [pyzo/pyzo#658]
pyzo version 4.9 claims to be version 0.0.0:

<img width="221" alt="Screenshot 2019-12-02 at 19 49 31" src="https://user-images.githubusercontent.com/1261954/69986070-dd1b5b00-153c-11ea-8622-a31492a9054e.png">

**Comment by almarklein** [comment]
Thanks for reporting this. That looks rather ugly.

**Comment by almarklein** [comment]
I looked into this a bit, but so far have not found how/where I can add a version string when building the app.

**Comment by core-code** [comment]
i'm not sure why the     "version=version," line in setup.py doesn't already do the right thing

i guess there are at least two options, either specify the Info.plist for the setup:

```

    OPTIONS = {
        "plist": {
            "CFBundleDisplayName": "pyzo",
            "CFBundleExecutable": "pyzo",
            "CFBundleIconFile": "pyzologo.icns",
            "CFBundleIdentifier": "pyzo",
            "CFBundleName": "pyzo",
            "CFBundleVersion": version,
            "CFBundleShortVersionString": version,
            },
    }



    setup(.... options={"py2app": OPTIONS}... )

```

alternatively you can always post-process the Info.plist inside the finished app bundle with PlistBuddy
`/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString $VERSIONSTRING" "${PROJECT_DIR}/${INFOPLIST_FILE}"
/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $VERSIONSTRING" "${PROJECT_DIR}/${INFOPLIST_FILE}"
`

**Comment by almarklein** [comment]
We're using PyInstaller, not py2app :)  but I added a post-processing step to the freeze script to add the plist options in #675

**Comment by core-code** [comment]
thanks!

**Comment by core-code** [comment]
i can confirm this is fixed in 4.10.2

**Comment by almarklein** [comment]
Thanks for confirming! 🎆
